Omikron/Factfinder/Helper/Data.php

```diff
@@ -130,12 +142,12 @@ public function getShowAddToCartButton()
 
     /**
      * Returns the current FACT-Finder version
-     * @param null|string $storeCode
+     * @param null|int|string $scopeCode
```

Comment:
Why did you rename it to $scopeCode when you pass the store id as parameter? The previous doc name did fit better to the current argument from line 85 Communication.php. 

Also $scopeCode on the other methods.